configure.ac

```diff
@@ -362,6 +362,14 @@ AS_IF([test "$enable_man_html" = yes && test "$enable_man" = no], [
 ])
 AM_CONDITIONAL(ENABLE_MAN_HTML, test "$enable_man_html" = yes)
 
+AC_ARG_ENABLE(boot-count,
```

Comment:
I think instead of a build-time option this should be a runtime option in the sysroot; something like 
```
[sysroot]
boot-counting = true
```
or so?

Reply:
yes, makes sense, will fix. thanks!

Reply:
fixed